satellite/marketing/marketingweb/server.go

```diff
@@ -16,7 +17,10 @@ import (
 )
 
 // Error is satellite marketing error type
-var Error = errs.Class("satellite marketing error")
+var (
+	Error     = errs.Class("satellite marketing error")
+	templates *template.Template
```

Comment:
This will have race condition, shouldn't be global.

Reply:
Good point. I changed it to be on `Server`